static-build/pages/about/index.tsx

```diff
@@ -60,7 +61,12 @@ const AboutPage: FunctionalComponent<Props> = () => {
             </div>
           </section>
         </header>
-        <main dangerouslySetInnerHTML={{ __html: ABOUT }} />
+        <main>
+          <div
+            class={`${$contentContainer} ${$aboutContent}`}
+            dangerouslySetInnerHTML={{ __html: ABOUT }}
+          />
+        </main>
```

Comment:
The main problem we had previously is `<main>` set the width of the page, but then inner elements wanted to 'break out' of that width, so were doing negative margin hacks, and relying on `100vw` to break back out to full width. `100vw` doesn't cater for the scrollbar, so we got horizontal scrolling (unless you were using overlay scrollbars on mac, which is why none of us saw it).

I've tried to untangle it a bit, so the width-setting stuff is just a class that can be applied to a div (or some other more semantic element). This means it can be closed, so another element on the page can become full width.